apps/backend/src/resolvers/types/ProposalStatusActionConfig.ts

```diff
@@ -47,6 +52,9 @@ export class EmailStatusActionRecipientsWithTemplate {
 
   @Field(() => EmailStatusActionEmailTemplate)
   emailTemplate: EmailStatusActionEmailTemplate;
+
+  @Field(() => [String], { nullable: true })
+  otherRecipientEmails: string[];
```

Comment:
If it is a nullable property, then the otherRecipientEmails should be marked as optional with a "?:" operator, shouldn't it?